docs/QbeastFormat.md

```diff
@@ -18,59 +18,70 @@ Following each write transaction is the creation of a new log file. **Table-leve
+Let's take a look to the Index metadata to understand what is written in the Delta Log JSON files.
 
-### Components of the index
+### Index Metadata
 
-On a high level, the index consists of one or more `OTrees` that contain `cubes`(or nodes), and each cube is made of `blocks` that contain the actual data written by the user. All records from the log are of **block/file-level** information.
+On a high level, the index consists of one or more `OTrees` that contain `cubes` (or nodes), and each cube is made of `blocks` that contain the actual data written by the user.
+All records from the log are of **block/file-level** information.
 
-- `revision` locates the particular tree
+| Term              | Description                                                                                                                                                                                                                                                                                               |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `revision`        | Locates the particular tree in the space, which includes the min/max values for each column indexed and timestamp.                                                                                                                                                                                |
+| `transformations` | Inside `revision` consist of two maps (in this case), each corresponding to one of the `indexedColumns`. Each pair of `min`/`max` defines the range of values of the associated indexed column that the `tree` can contain and is to be expanded to accommodate new rows that fall outside the current range. |
+| `cube`            | Identifies the current `block`'s `cube` from the `tree`.                                                                                                                                                                                                                                                  |
+| `state`           | Of the `block` determines the **READ** and **WRITE** protocols. It can be `FLOODED`, `ANNOUNCED`, or `REPLICATED`.                                                                                                                                                                                        |
+| `weightMax/weightMin` | Each element gets assigned with a uniformly created `weight` parameter. `weightMin` and `weightMax` define the range of weights that the `block` can contain.                                                                                                                                             |
```

Comment:
`minWeight`, `maxWeight`(?)

Reply:
I would delete this table because the info is duplicated, sorry for the inconvenience.